src/bin/vip-import-sql.js

```diff
@@ -190,44 +191,44 @@ export async function gates( app, env, fileMeta ) {
 const examples = [
 	// `sql` subcommand
 	{
-		usage: 'vip import sql @mysite.develop file.sql',
-		description: 'Import the given SQL file to your site',
+		usage: 'vip @example-app.develop import sql file.sql',
+		description:
+			'Import the local SQL database file "file.sql" to the develop environment of the "example-app" application.',
```

Comment:
"local SQL database file" doesn't sound right to me. I asked ChatGPT about alternative terms for SQL dump, a common term for the file the VIP tooling expects to receive, and it provided some suggestions:

> A good term to describe the output of a SQL database dump is **"database snapshot"** or **"database export"**. Both terms refer to a complete capture of the database's current state, typically used for backups or migrations. Other options include:
> 
> - **SQL dump file**: Referring to the file format itself.
> - **Database backup**: If the dump is intended as a backup.
> - **Data export**: Emphasizing that it contains exported data for use elsewhere.

I think "local SQL data export file" or "local SQL database backup file" might fit here. If you're open to that change, we'd want to apply it to several other places in the PR to stay consistent.

Reply:
Valid! Making this update in a new commit.